### DISCUSSION TITLE
demo videos: Snap panel, Window Throw, Stash

### DISCUSSION THREAD

**Discussion by pinkgalaxy** [rxhanson/RectanglePro-Community#590] [Show and tell]
Snap panel 
https://github.com/rxhanson/RectanglePro-Community/assets/164553001/d10a648e-fc89-40f4-ba89-340584ac5a9f

Window Throw 
https://github.com/rxhanson/RectanglePro-Community/assets/164553001/91947692-6db6-40c9-bafc-e6889b1c3827

Stash 
https://github.com/rxhanson/RectanglePro-Community/discussions/422#discussioncomment-8889036

**Comment by rxhanson** [comment]
Sorry for the delayed response, but thanks for these!

**Comment by BrentEubanks** [comment]
The video above shows Quick Throw but not basic Window Throws.  
Here's a video that shows the basic Window Throw: https://rectangleapp.com/comparison

It took be quite a bit of searching to find (whereas this one popped up right away) so I thought I'd leave this here for someone else's future reference.
